packages/react-native/src/components/QuilttConnector.tsx

```diff
@@ -28,6 +28,7 @@ export type PreFlightCheck = {
   error?: string
 }
 
+// We may be able to remove this since the server will render an error page
```

Comment:
Defense in layers. Everything is always changing.

Reply:
makes sense - going to remove the comment for now